Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml

```diff
@@ -652,7 +672,48 @@ tasks:
       {
         "position": {
           "x": -200,
-          "y": 2100
+          "y": 2090
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "69":
+    id: "69"
+    taskid: 437703be-f9d0-42eb-864c-de6fd5f869e7
+    type: playbook
+    task:
+      id: 437703be-f9d0-42eb-864c-de6fd5f869e7
+      version: -1
+      name: Cortex ASM - Azure Enrichment
+      description: Given the IP address this playbook enriches Azure information relevant to ASM alerts.
+      playbookName: Cortex ASM - Azure Enrichment
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      RemoteIP:
+        complex:
+          root: inputs.RemoteIp
```

Comment:
Change this to: inputs.RemoteIP
this is how it is specified in playbook inputs and causing 2nd error in https://app.circleci.com/pipelines/github/demisto/content/231876/workflows/b391dc0e-22a8-44d0-ae08-cae7d5bd3d71/jobs/650542

Reply:
resolved in [Update input for Cortex ASM playbook](https://github.com/demisto/content/pull/24310/commits/40d107b39cd4653abe8e2fb89cb07126d9d6dcb5)